frontend/src/App.tsx

```diff
@@ -1,9 +1,18 @@
+import { useState } from 'react';
 import './App.css';
+import { Post, Header, Profile, CreatePost } from './components';
 
 function App() {
+  const [showCreatePostModal, setShowCreatePostModal] = useState(false);
+
   return (
-    <div>
-      <h1>Lets go!</h1>
+    <div className="app-body">
+      <Header onCreatePostClick={() => setShowCreatePostModal(true)} />
+      {showCreatePostModal && <CreatePost setCreatePostBox={setShowCreatePostModal} />}
+      <Post name="Ejaz hussain" number={3} />
```

Comment:
a better approach would be to store these instances of static inputs in a dummy object so its easier to refactor later when you integrate with backend. otherwise it might miss some things